### PR TITLE
Updates hyrax views #1864

### PR DESCRIPTION
**Pull request by alexBLR**

### CODE REVIEW

File: app/views/hyrax/base/_show_actions.html.erb [hunk 1]
```diff
@@ -1,12 +1,13 @@
-<% # [Hyrax-overwrite-v3.0.0.pre.rc1] Only display delete button if user is an admin L#36 %>
+<% # [Hyrax-overwrite-v3.4.1] Only display delete button if user is an admin L#36 %>
 <% # Do not display feature button L#10 (removed default code) %>
 <div class="row show-actions button-row-top-two-column">
   <div class="col-sm-6">
-    <% if presenter.show_deposit_for?(collections: @user_collections) && current_user.admin? %>
+    <% if !workflow_restriction?(presenter) %>
+    <% if presenter.show_deposit_for?(collections: @user_collections) %>
```
Comment (bwatson78) [inline app/views/hyrax/base/_show_actions.html.erb:6]: Can you combine together the test methods? Like
`if !workflow_restriction?(presenter) && presenter.show_deposit_for?(collections: @user_collections)`

File: app/views/hyrax/base/show.html.erb [hunk 2]
```diff
@@ -63,5 +64,10 @@
     <%# = render 'sharing_with', presenter: @presenter %>
     <%# = render 'user_activity', presenter: @presenter %>
 
-  </div>
+    <span class='hide analytics-event' data-category="work" data-action="work-view" data-name="<%= @presenter.id %>" >
+    <% @presenter.member_of_collection_ids.each do |collection_id| %>
+      <span class='hide analytics-event' data-category="work-in-collection" data-action="work-in-collection-view" data-name="<%= collection_id %>" >
+    <% end %>
+  
+    </div>
```
Comment (bwatson78) [inline app/views/hyrax/base/show.html.erb:72]: The spacing needs to be reduced here.

File: app/views/hyrax/collections/_show_descriptions.html.erb [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
     <div class='col-xs-12 col-sm-8'><dd><a href="<%= @presenter.purl %>" target="_blank"><%= @presenter.purl %></a></dd></div>
   </div>
   <!-- Persistent URL block end -->
-    <% present_terms(@presenter, @presenter.terms_with_values) do |r, term| %>
-      <div class='metadata-line'>
-        <div class='col-xs-12 col-sm-4'><dt><%= r.label(term) %></dt></div>
-        <div class='col-xs-12 col-sm-8'><dd><%= r.value(term) %></dd></div>
+    <% @presenter.terms_with_values.each do |field_name| %>
+          <div class='metadata-line'>
+        <div class='col-xs-12 col-sm-4'><dt><%= collection_metadata_label(@presenter, field_name) %></dt></div>
```
Comment (bwatson78) [inline app/views/hyrax/collections/_show_descriptions.html.erb:11]: Line# 10 should be in the same column as 11.

Comment (bwatson78) [reply app/views/hyrax/collections/_show_descriptions.html.erb:11]: Oops, L#10 should just return to where it was.

Comment (bwatson78) [reply app/views/hyrax/collections/_show_descriptions.html.erb:11]: @alexBLR This is the only one still needing attention.

Comment (alexBLR) [reply app/views/hyrax/collections/_show_descriptions.html.erb]: 👍

**Review by bwatson78 — APPROVED** [review]

**Pull request merged**